### PR TITLE
Update to share consent with Unity Ads only when user provides an explicit yes/no. In all other cases, Unity Ads SDK will collect its own consent per the guidelines [here][https://unity3d.com/legal/gdpr]

### PR DESCRIPTION
**Pull request by ekrzanich**
…es or no

### CODE REVIEW

File: UnityAds/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 ## Changelog
+  * 2.2.1.2
+    * Pass explicit consent to Unity Ads (with revision).
```
Comment (rsatrasala) [inline UnityAds/CHANGELOG.md]: For parity with iOS ""Update to share consent with Unity Ads only when user provides an explicit yes/no. In all other cases, Unity Ads SDK will collect its own consent per the guidelines [here][<relevant documentation link here>]""

**Comment by rsatrasala** [comment]
@ekrzanich Changes look good and I have same question about DNT here as well.

File: UnityAds/CHANGELOG.md (hunk 1 above)
Comment (ekrzanich) [reply UnityAds/CHANGELOG.md]: Update to share consent with Unity Ads only when user provides an explicit yes/no. In all other cases, Unity Ads SDK will collect its own consent per the guidelines [here][https://unity3d.com/legal/gdpr]

**Comment by ekrzanich** [comment]
We don't handle LAT differently in terms of GDPR

**Pull request merged**